app/controllers/schools/estimate_participants_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class Schools::EstimateParticipantsController < Schools::BaseController
+  # who has permissions to view this ?
+  skip_after_action :verify_authorized
+  skip_after_action :verify_policy_scoped
+  before_action :get_school_cohort
+
+  def edit
+    @school_cohort_form = SchoolCohortForm.new
+  end
+
+  def update
+    @school_cohort_form = SchoolCohortForm.new(form_params)
+
+    if @school_cohort_form.valid?
+      @school_cohort.update!(form_params)
+      redirect_to "/schools/cohorts/2021" # TODO: this needs changing?
+    else
+      render :edit
+    end
+  end
+
+private
+
+  def get_school_cohort
+    @school_cohort = SchoolCohort.find(params[:id])
```

Comment:
You'll need to select the school_cohort for the correct school, too - see here: https://github.com/DFE-Digital/early-careers-framework/blob/develop/app/controllers/schools/cohorts_controller.rb#L7-L19

Reply:
It depends what params[:id] came from. If it is the id of the school cohort then this will work

Reply:
I am getting the id param from the previous screen, /schools/cohort/2021, would that not identify the correct SchoolCohort?

Reply:
ahh, that's true - the other URLs are using the start_year, not the ID

Reply:
That is what I am doing, the id is something like adsd-afafsg-435345-43645654 .  Could we not slug the /schools/cohort/:start_year instead of using a member route?

Reply:
> That is what I am doing, the id is something like adsd-afafsg-435345-43645654 . Could we not slug the /schools/cohort/:start_year instead of using a member route?

i was hoping someone else would reply to this but nobody else has 😆 what do you mean? I don't really understand

Reply:
https://medium.com/@hartaniyassir/creating-slug-urls-in-rails-without-gems-c693e0eeec8a

Reply:
ah cool, didn't know that was a thing! yeah, that seems like a good idea